CMakeLists.txt

```diff
@@ -796,6 +800,7 @@ target_link_libraries (seastar
     StdAtomic::atomic
     lksctp-tools::lksctp-tools
     rt::rt
+    ucontext:ucontext
```

Comment:
@a16bitsysop you might need to put `ucontext::ucontext` (two colons). but i created #1799, as i think there is no need to use a dedicated source file for checking the availability of the `getcontext()` function. could you please give it a try ?

Reply:
thanks, dropped that commit and tested yours